### PR TITLE
Will be putting this on my resume

### PR DESCRIPTION
**Pull request by blakeparkinson**

### CODE REVIEW

File: routes/leaguesearch.js [hunk 1]
```diff
@@ -4,125 +4,144 @@ var http = require("http");
...
+        { popular_teams: results.goat_lists }
+    );  
+  })
+
+});
+
+var fetchLeadersLists = function(callback){
+  db.collection('leaders').find().toArray(function (err, items){
+
+    callback(null, items);
```
Comment (kjgross) [inline routes/leaguesearch.js]: why this first null term?

File: routes/leaguesearch.js [hunk 2]
```diff
@@ -4,125 +4,144 @@ var http = require("http");
...
-})
+
+  async.parallel({
+    goat_lists: fetchGoatLists,
+    leaders_lists: fetchLeadersLists
+  }, function(err,results){
+      console.log(results.goat_lists);
+      res.render('leaguesearch',
+        { popular_teams: results.goat_lists }
+    );
```
Comment (kjgross) [inline routes/leaguesearch.js]: Are we not rendering the leaders list stuff just at this time or does it go somewhere else?

**Comment by kjgross** [comment]
Perrrrty!

But where would I see the goat lists on FE? not seeing anything on the /leaguesearch page for it

File: routes/leaguesearch.js (hunk 1 above)
Comment (blakeparkinson) [inline routes/leaguesearch.js]: that's what the callback expects, first param error, 2nd param is the data

File: routes/leaguesearch.js (hunk 2 above)
Comment (blakeparkinson) [inline routes/leaguesearch.js]: We are. Im gonna do it now

**Comment by blakeparkinson** [comment]
@kjgross kilmgee. It's working now, I'll do the front end for rendering the leaders stuff. Can you add a limit param to all these functions? Default is 10?

 the query to leaders is a little slow tho. Let's investigate why. Maybe we can look into a caching module

**Pull request merged**